src/Transpiler.ts

```diff
@@ -1197,7 +1200,7 @@ export abstract class LuaTranspiler {
         if (!ts.isPropertyAccessExpression(node.expression)
             && !ts.isElementAccessExpression(node.expression)
             && !tsHelper.getCustomDecorators(type, this.checker).has(DecoratorKind.NoContext)) {
-            params = this.transpileArguments(node.arguments, ts.createIdentifier("_G"));
+            params = this.transpileArguments(node.arguments, ts.createIdentifier(this.isStrict ? "nil" : "_G"));
```

Comment:
Maybe instead of creating an identifier you could just create a null or undefined keyword? I don't know how big that change would be

Reply:
The context paramater of transpileArguments is optional and skips adding the initial parameter if omitted (or undefined). I could re-purpose null to explicitly convert to nil but having separate logic for null and undefined always feels icky to me.

I could change transpileArguments to take either an expression or a string as the context parameter, and just pass the "nil" instead of a temp identifier.

Reply:
I don't mean actual null, I meant something like `ts.CreateNullKeyword()` or something similar.

Reply:
actually nvm that won't work here since it is an abstract argument and not a concrete instantiation of one.

Reply:
Ah - sorry - misunderstood. Yes, that seems like a good idea.